### PR TITLE
Support for VLAN feature.

### PR DESCRIPTION
**Pull request by justinejose91**
1. VLAN Create and Delete
2. Adding / removal tagged/untagged ports.
3. #show Vlan, #show Vlan <id> using sonic-yang.
4. Configuring / Removing MTU / admin-status
5. Added the sonic-vlan yang

### CODE REVIEW

File: models/yang/sonic/sonic-portchannel.yang [hunk 1]
```diff
@@ -0,0 +1,77 @@
+module sonic-portchannel {
```
Comment (jeffyin-dell) [inline models/yang/sonic/sonic-portchannel.yang:1]: Please fix the indentation in this file to 4 spaces per tab

File: src/CLI/clitree/cli-xml/interface.xml [hunk 2]
```diff
@@ -399,24 +409,68 @@ limitations under the License.
...
-            <COMMAND name="no sflow sampling-rate" help="set sampling rate to default" ptype="SUBCOMMAND" mode="subcommand">
-                <PARAM name="rate" help="sampling rate" ptype="UINT"/>
-            	<ACTION builtin="clish_nop"/>
-            </COMMAND>
-            <COMMAND name="no sflow enable" help="disable sFlow" ptype="SUBCOMMAND" mode="subcommand">
-            	<ACTION builtin="clish_nop"/>
+
+       <COMMAND
+           name="sflow"
+           help="interface sFlow settings"
```
Comment (jeffyin-dell) [inline src/CLI/clitree/cli-xml/interface.xml]: Since you're already editing the sFlow commands, can you capitalize the help strings for these commands?

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]

File: src/CLI/clitree/cli-xml/interface.xml (hunk 2 above)
Comment (justinejose91) [reply src/CLI/clitree/cli-xml/interface.xml]: Yes Jeff, I addressed it. These commands were not written in the format we do follow in interface.xml, that's the reason I changed it. Anyway, thanks for that catch :)

File: models/yang/sonic/sonic-portchannel.yang (hunk 1 above)
Comment (justinejose91) [reply models/yang/sonic/sonic-portchannel.yang:1]: Done!

File: src/CLI/actioner/sonic-cli-if.py [hunk 3]
```diff
@@ -182,10 +197,6 @@ def run(func, args):
 
     try:
         # Temporary code for #show vlan command with dummy data
```
Comment (jeffyin-dell) [inline src/CLI/actioner/sonic-cli-if.py:199]: You can delete this line as well

File: src/translib/intf_cmn.go [hunk 4]
```diff
@@ -0,0 +1,990 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//////////////////////////////////////////////////////////////////////////
+
+package translib
+
+import (
```
Comment (jeffyin-dell) [inline src/translib/intf_cmn.go:21]: Should fix the tabbing in this file as well.

File: src/translib/intf_utils.go [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//////////////////////////////////////////////////////////////////////////
+
+package translib
```
Comment (jeffyin-dell) [inline src/translib/intf_utils.go:19]: Tabs should be 4 spaces

File: src/translib/vlan_intf.go [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//////////////////////////////////////////////////////////////////////////
+
+package translib
+
+import (
```
Comment (jeffyin-dell) [inline src/translib/vlan_intf.go:21]: Fix tabbing in this file as well.

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Sorry, more nitpicking on tabs

**Review by jeffyin-dell — APPROVED** [review]

**Pull request merged**